src/test/features/claim/routes/task-list.ts

```diff
@@ -23,14 +24,41 @@ describe('Claim issue: task list page', () => {
     checkAuthorizationGuards(app, 'get', ClaimPaths.incompleteSubmissionPage.uri)
     checkEligibilityGuards(app, 'get', ClaimPaths.incompleteSubmissionPage.uri)
 
-    it('should render page when everything is fine', async () => {
+    it('should render page when everything is fine when user role present', async () => {
       idamServiceMock.resolveRetrieveUserFor('1', 'citizen')
       draftStoreServiceMock.resolveFind('claim')
+      claimStoreServiceMock.resolveRetrieveUserRoles('cmc-new-features-consent-given')
 
       await request(app)
         .get(ClaimPaths.taskListPage.uri)
         .set('Cookie', `${cookieName}=ABC`)
         .expect(res => expect(res).to.be.successful.withText('Make a money claim'))
     })
+
+    it('should show error page when user role cannot be retrieved', async () => {
+      idamServiceMock.resolveRetrieveUserFor('1', 'citizen')
+      draftStoreServiceMock.resolveFind('claim')
+      claimStoreServiceMock.rejectRetrieveUserRoles()
+      await request(app)
+        .get(ClaimPaths.taskListPage.uri)
+        .set('Cookie', `${cookieName}=ABC`)
+        .expect(res => expect(res).to.be.serverError.withText('error'))
+    })
+  })
+
+  describe('on GET', () => {
```

Comment:
Please remove as we already have `describe('on GET', () => {`, see line 22. Please do the same to lines 50 and 51 as this is just duplicate.